ddtrace/internal/module.py

```diff
@@ -53,6 +63,19 @@ def _patch_run_code() -> None:
         runpy._run_code = _wrapped_run_code  # type: ignore[attr-defined]
 
 
+def register_run_module_transformer(transformer: TransformerType) -> None:
+    """Register a run module transformer."""
+    _run_module_transformers.append(transformer)
+
+
+def unregister_run_module_transformer(transformer: TransformerType) -> None:
+    """Unregister a run module transformer.
+
+    If the transformer was not registered, a ``ValueError`` exception is raised.
+    """
+    _run_module_transformers.remove(transformer)
+
+
```

Comment:
I think this is just dead code for now. This would be useful for a stand-alone coverage tool whereby we would have to make sure that we instrument the main module that's being run. With pytest we don't have this problem.

Reply:
🤔 , what about in the `ddtrace-run` context? eg: with our `unittest` integration?

Reply:
What would the main module be in that case? Is it not `unittest` itself? If so, I think we wouldn't need this code

Reply:
I think it depends (eg: someone could have a wrapper that loads the class and does some discovery).

I guess I'll cross that bridge when I get to it. I removed it in 05b2686fd .